docs/CODEOWNERS

```diff
@@ -1,7 +1,2 @@
-# -*- mode: gitignore; -*-
```

Comment:
@MXWest I hadn't seen this before. Did you want it here on purpose?

Reply:
I honestly don't know what that is.

Reply:
I think it's there for code editors to understand that CODEOWNERS has the same syntax as `gitignore` for code highlighting.